orc8r/cloud/go/tools/combine_swagger/main.go

```diff
@@ -48,7 +49,11 @@ func main() {
 	}
```

Comment:
I think this would be a good addition to the tool. But since the end-goal of Andy's project is to deprecate + eventually remove the need for this tool, not worth updating it at this point

Reply:
I personally trend away from defaults, and particularly in this case

- The `combine_swagger` tool is only called from one location in our codebase, so there's not a strong need to add a default
- Adding a default adds an additional layer of complexity when trying to debug and/or assert correctness
- This tool's main job after Andy's project concludes will be to create the "single static" swagger.yml we have checked into vcs. So since its functionality will be minimal / not necessary for correctness, probably not worth updating the tool with new functionality at this point